CRM/Core/Error.php

```diff
@@ -945,6 +945,15 @@ public static function createError($message, $code = 8000, $level = 'Fatal', $pa
     return $error;
   }
 
+  /**
+   * Create a random identifier for an error.
+   *
+   * @return string
+   */
+  public static function createErrorId(): string {
+    return rtrim(chunk_split(CRM_Utils_String::createRandom(12, CRM_Utils_String::ALPHANUMERIC), 4, '-'), '-');;
```

Comment:
why the double `;` @totten

Reply:
Emphasis? Sometimes the interpreter doesn't listen. :wink: 

---

Thanks, I'll push up fix.